src/actions/guides/command_line_tasks/custom_tasks.cr

```diff
@@ -11,9 +11,9 @@ class Guides::CommandLineTasks::CustomTasks < GuideAction
 
     Place custom tasks in the `tasks` folder of your application.
     Your custom task must:
-    1. Inherit from `LuckyCli::Task`
-    2. Implement a `call` method
-    3. Include a `summary`
+    * Inherit from `LuckyCli::Task`
+    * Implement a `call` method
+    * Include a `summary`
```

Comment:
This was because they were being displayed all weird and out of alignment. I didn't feel like fighting css 😛

Reply:
Haha sounds good